### DISCUSSION TITLE
Support for front attached leveling blade.

### DISCUSSION THREAD

**Discussion by Cosette14234** [Courseplay/courseplay#7453] [Ideas]
A front attached leveling blade support would be nice ( i personally would like to use a buldozer ( DT-75 ) to push the chaff to the back of the bunker, but it either isnt supported or i dont know how to set it up. If it isnt supported ( wich is what im guessing it is ) then i hope that it will get supported some time soon.

**Comment by Tensuko** [comment]
Would have to change to much, wont happen.

**Reply by Cosette14234** [reply]
Well, it was worth asking atleast :-/
